apps/next-product-starter/src/pages/buy.tsx

```diff
@@ -17,7 +28,23 @@ const Buy: FunctionComponent<BuyProps> = ({bundles}) => {
   return (
     <Layout meta={{title: `Buy ${config.defaultTitle}`}}>
       <ClaimCoupon sellable={sellable} />
-      <Commerce bundles={bundles} />
```

Comment:
instead of a single `Commerce` component here I broke it down into part, similar to the work we've poked at with `Card` for egghead next.

The result is more composable, so technically we can hot swap aspects of the checkout widget more readily and should make conversations around customizations easier to have.

This was a quick take on this, and I think it could use more thought and attention.